notebooks_tsqr/NightLog.ipynb

```diff
@@ -27,7 +27,7 @@
     "day_obs = \"2024-09-25\"  # TODO Change to 'YESTERDAY' to test with default before push\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    if allsrc.exp_src.exposures["LATISS"]:
</pre><p>This is not a TODO to fix on this PR but me noticing where we need to clarify: </p><p>I am reading as this to only apply to latiss, but I think these portions where in the storyboard we have 'LATISS' as 'This selected instrument, latiss being one of them' </p><p>Would you agree that this section would be valid for any of the instruments? </p><p>I think we just need to implement that larger selectable instrument part still. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_logging_and_reporting/pull/28/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>